.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,6 +1,7 @@
 ---
 name: Feature Request
 about: Suggest a new feature
+explanation: Explain your new feature idea
```

Comment:
Not really sure if `explanation` is a correct front matter variable. Can you like a resource for more information on this?